test-clients/src/main/java/com/hedera/services/bdd/suites/token/TokenManagementSpecs.java

```diff
@@ -114,6 +115,7 @@ public HapiApiSpec wipeAccountSuccessCasesWork() {
 	public HapiApiSpec wipeAccountFailureCasesWork() {
```

Comment:
You need to update the `WipeAccountSuccessCasesWork` test as-well.
Currently, the only thing that is being checked is:
- `misc` balance is reduced by 500 (wiped amount)
- `tokenTreasury` balance is the same
There is another effect from the token wipe -> the `total supply` or `tokenFloat` is reduced. You need to check that.